py4DSTEM/visualize/overlay.py

```diff
@@ -225,7 +225,7 @@ def add_ellipses(ax,d):
     Parameters:
         center
         a
-        e
+        b
```

Comment:
I cannot figure out how to comment above, but plt is an unused import.

Reply:
And not the greatest docstring here.

Reply:
Updates to rm `plt`.  The docstrings in `overlay` all suck, and should all be updated.....but I'm gonna call this out of scope for now, as I'd rather not update just the docstring for this one overlay fn, and doing all of them is def out of scope.  So agreed, we (prob me...!) should return to this